main.go

```diff
@@ -11,6 +11,7 @@ import (
 	"net/http"
 	"os"
 	"os/signal"
+	"slices"
```

Comment:
The last time I was writing much Golang, it didn't yet have a built in way to check if a value was already in an array.  The `slices` module finally adds that, in the August 2023 language update.